mi/dataset/parser/ctdbp_cdef_dcl.py

```diff
@@ -51,7 +51,9 @@
 
 from mi.dataset.dataset_parser import SimpleParser
 
-from mi.dataset.parser.utilities import dcl_controller_timestamp_to_utc_time
+from mi.dataset.parser.utilities import  \
+    dcl_controller_timestamp_to_ntp_time, \
+    timestamp_ddmmyyyyhhmmss_to_ntp_time
```

Comment:
no because there are many different type of time_string, exammple,  "yyyymmdd HH:MM:SS"
different than "yymmdd HH:MM:SS: , different than "mmddyyyy"